support_scripts_perl/load_gtf_ensembl.pl

```diff
@@ -117,7 +117,7 @@
 
 if($load_type eq 'gene') {
   load_genes($dba,$gtf_file,$slice_hash,$analysis,%strand_conversion);
-} elsif($load_type eq 'single_line_feature' and ($analysis_name eq 'dust' or $analysis_name eq 'repeatdetector' or $analysis_name eq 'trf')) {
+} elsif($load_type eq 'single_line_feature' and ($analysis_name eq 'dust' or $analysis_name eq 'repeatdetector' or $analysis_name eq 'trf' or $analysis_name eq 'repeatmask_repbase_human')) {
```

Comment:
So loading repeatmasker output will only work for the human library so far? 
We should consider making this more generic, as we would like to move forward with the repeats pipeline where we will run repeatmasker with the RepeatModeler generated libraries (analysis_name=>'repeatmask_repeatmodeler'), and we will also want to be able to call the anno implementation to run with all repbase libraries, not just human. 
This can be in another PR though (should probably be devloped as part of the repeat pipeline).